pkgs/tools/security/pass/default.nix

```diff
@@ -1,6 +1,7 @@
 { stdenv, lib, pkgs, fetchurl, buildEnv
 , coreutils, gnused, getopt, git, tree, gnupg, which, procps, qrencode
 , makeWrapper
+, openssl ? stdenv.isDarwin
```

Comment:
This doesn't make sense, it defines a boolean as default value for the `openssl` argument - which is a derivation, not a boolean.  Just `openssl` should be fine.

Reply:
Ah, I see, thanks for the comment!